### PR TITLE
the one that adds context to summaries 

### PR DESCRIPTION
**Pull request by sturobson**
some tweaks to update for 'real world usage' and added the `{% if context %}` for each variant.

A first step to a single component to rule them all.

### CODE REVIEW

File: components/vf-summary/vf-summary.config.yml [hunk 1]
```diff
@@ -31,3 +86,5 @@ variants:
     hidden: true
     context:
       size: s
+  - name: event-easy
```
Comment (khawkins98) [inline components/vf-summary/vf-summary.config.yml:89]: Is this intended to be a new variant?

Comment (sturobson) [reply components/vf-summary/vf-summary.config.yml:89]: It was, an idea from the before times that never got into being used. So I’ve hidden it here. 

https://latest.visual-framework.dev//components/vf-summary/index.html?q=Summary#vf-summary--event-easy

Comment (khawkins98) [reply components/vf-summary/vf-summary.config.yml:89]: ah, didn't realise that was live. Yeah, the event container. Was a nice design from Mark, but would probably need an overhaul given updated context.

**Comment by sturobson** [comment]
I've a separate update to this that will suggest adding `role="presentation"` after `alt=""` if you are using an image but don't want to describe it for a screen reader.

This would be useful for training as the images used are quite abstract. 

For news - it would be a suggestion to always add good descriptive text (we can but hope).

**Review by khawkins98 — APPROVED** [review]

**Pull request merged**